docs/demos/tutorials/03_Blocking.ipynb

```diff
@@ -794,6 +794,17 @@
    "source": [
     "linker.profile_columns(\"city || left(first_name,1) \")"
    ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "!!! note \"Further Reading\"\n",
+    "    :material-tools: For more on the blocking tools in Splink, please refer to the [Blocking API documentation](../../linkerbloc.md).\n",
+    "\n",
+    "    :simple-readme: For a deeper dive on blocking, please refer to the [Blocking Topic Guides](../../topic_guides/blocking_rules.md)."
```

Comment:
I'd be tempted to swap these. Background on blocking is likely more beneficial for beginners.